geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSInterIntegrationTest.java

```diff
@@ -30,13 +32,17 @@
 import redis.clients.jedis.JedisCluster;
 import redis.clients.jedis.Protocol;
 
+import org.apache.geode.redis.ConcurrentLoopingThreads;
 import org.apache.geode.redis.RedisIntegrationTest;
 import org.apache.geode.test.awaitility.GeodeAwaitility;
 
 public abstract class AbstractSInterIntegrationTest implements RedisIntegrationTest {
   private JedisCluster jedis;
   private static final int REDIS_CLIENT_TIMEOUT =
       Math.toIntExact(GeodeAwaitility.getTimeout().toMillis());
+  private static final String SET1 = "{user1}set1";
+  private static final String SET2 = "{user1}set2";
+  private static final String SET3 = "{user1}set3";
```

Comment:
Could these (and other instances of the `{user1}` tag in this class) be renamed to `{tag1}` etc. to preserve the changes from GEODE-9932?